README.md

```diff
@@ -89,13 +98,15 @@ Allows you to release updates to your iOS TestFlight beta app or production app
 
 1. **What to Test?** *(File path)* - Path to the file containing notes on what to test for this release.
 
-2. **Skip Build Processing Wait** *(Checkbox)* - Skip waiting for App Store to finish the build processing.
+2. **App specific apple Id** *(String)* - App specific apple Id allows you to upload applications to a TestFlight track without triggering 2FA. This value should be taken from Apple ID property in the App Information section in App Store Connect.
```

Comment:
Nit: I assume it will be better to maintain name convection and rename this option to `App Specific Apple Id`

Reply:
Yep, thanks - fixed it